content/docs/command-reference/run.md

```diff
@@ -228,13 +228,12 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   `command`.
 
 - `--no-exec` - create a stage file, but do not execute the `command` defined in
-  it, nor cache dependencies or outputs (like with `--no-commit`, explained
-  below). DVC will also add your outputs to `.gitignore`, same as it would do
-  without `--no-exec`. Use `dvc commit` to force committing existing output file
-  versions to cache.
+  it. DVC will still add the outputs to `.gitignore`. Implies `--no-commit`
```

Comment:
It's too complicated - to go and read a reference ... then next sentence that you would get only if read the reference

Reply:
I'm not sure I follow this. By references here do you mean the --no-exec and --no-commit options?

Reply:
The alternative is to have duplicate explanation of --no-commit. Or maybe --no-commit should go before --no-exec — it's probably less rare actually.

Reply:
I removed the "Implies `--no-commit`" part for now, as it is, well... implied. No need to mention it, I guess (obviously since the stage didn't run no outputs were generated, etc.).